pkgs/by-name/ra/rapidraw/package.nix

```
@@ -0,0 +1,148 @@
+{
+  lib,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  rustPlatform,
+  makeWrapper,
+  pkg-config,
+  wrapGAppsHook4,
+  openssl,
+  onnxruntime,
+  webkitgtk_4_1,
+  gtk3,
+  glib,
+  gdk-pixbuf,
+  libappindicator,
+  cairo,
+  pango,
+  xorg,
+  libxkbcommon,
+  vulkan-loader,
+  libjpeg,
+  libpng,
+  zlib,
+  libGL,
+  dbus,
+  gvfs,
+  libheif,
+  glib-networking,
+  gsettings-desktop-schemas,
+  nodejs_20,
+  npmHooks,
+  cargo-tauri,
+}:
+
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
Rust builder now supports these kinda lambdas, prefer them over rec (there's job getting done to make them propagate earlier)